### ISSUE TITLE
Rooted successfully, no compatible devices found. 

### ISSUE THREAD

**Issue by vruzy** [fpv-wtf/wtfos-configurator#189]
I'm able to root the devices fine and receive the success message at the end.  Going to any other tab on the wftos page and clicking connect shows "no compatible devices found".   If I go back to the root'ing page the device shows available in com ports.  
 Same issue on v1 goggles, and caddx vista.

**Comment by stylesuxx** [comment]
Which firmware version are you on? Which browser, which os?

**Comment by vruzy** [comment]
.606
chrome
macos big sir

Shaun Ortman
***@***.***
(216)307-4286
www.treestreetstudio.com


On Oct 8, 2022, at 1:10 PM, Chris ***@***.***> wrote:

﻿

Which firmware version are you on? Which browser, which os?

**Comment by stylesuxx** [comment]
Please try the develop buil: https://develop.fpv.wtf/
If this does not work, jump on Discord  for a troubleshooting session

**Comment by stylesuxx** [comment]
@vruzy any update on this? Did it work with de development build?

**Comment by vruzy** [comment]
No updates.  Never got it working.
I did hop on discord and ask as well and got buried in a few hundred other messages.  One person replied and said rooting sometimes take several tries.

I have a paid fpv gig today I don't want to risk messing it up. So I'll try again this weekend.

Thanks,


Shaun Ortman
***@***.******@***.***>
treestreetstudio.com<http://treeststudios.com/>
[cid:0e49a853-346a-4ca4-97af-3979d31c0ead]<http://treeststudios.com/>
________________________________

**Comment by sebastianwachter** [comment]
I'm having the same issues:
- the root page finds my goggles (v2) as COM10
- the wtfos page show "no compatible devices found"

Goggles are on firmware version v01.00.0606
Browser is Google Chrome 106.0.5249.119 
OS is Windows 10

The development page (https://develop.fpv.wtf) sees my goggles in both the root and the wtfos page however I can't root there since the "signing server is unavailable" and the wtfos page show the goggles as "pigeon" (is this correct?) but connecting to it won't work with the adb error (adb is not running and I am not connected in another tab).

The root process on the https://fpv.wtf page repeatedly fails in step 3 and two times in step 4 (I tried rooting it multiple times). The console.log is: 

```
00:03 Attempting Step 1...
00:44  - Found Device: GP150, Version: 01.02.0000
00:44 Step 1 - Success! Rebooting...
01:07 Attempting Step 2...
01:16 Step 2 - Success!
01:19 Attempting Step 3...
01:21 Failed! Please power cycle your device now...
01:49 Attempting Step 2...
Unlock step failed: NoResponse: Step 2: get challenge
    at m.value (https://fpv.wtf/static/js/main.3c54cf69.js:2:1540004)
    at m.<anonymous> (https://fpv.wtf/static/js/main.3c54cf69.js:2:1547580)
    at l (https://fpv.wtf/static/js/main.3c54cf69.js:2:387588)
    at Generator._invoke (https://fpv.wtf/static/js/main.3c54cf69.js:2:387376)
    at Generator.next (https://fpv.wtf/static/js/main.3c54cf69.js:2:388017)
    at Fs (https://fpv.wtf/static/js/main.3c54cf69.js:2:768589)
    at a (https://fpv.wtf/static/js/main.3c54cf69.js:2:768793)
02:07 Step 2 - Success!
02:11 Attempting Step 3...
02:15 Step 3 - Success!
02:15 Attempting Step 4...
02:19 Unlock failed, please power-cycle your device and try again.
02:50 Device went away...
03:08 Device came back...
03:11 Attempting Step 4...
03:15 Unlock failed, please power-cycle your device and try again.
04:17 Device went away...
04:35 Device came back...
04:38 Attempting Step 4...
04:46 Step 4 - Success! Rebooting...
05:09 Attempting Step 5...
05:09 All done - your device has been successfully rooted!
```

**Comment by sebastianwachter** [comment]
The connection to from https://develop.fpv.wtf (the pigeon device) fails with:

```
Failed connecting to device: Error: Unknown error
    at e.<anonymous> (backend.ts:176:15)
    at l (runtime.js:63:40)
    at Generator._invoke (runtime.js:294:22)
    at Generator.next (runtime.js:119:21)
    at zs (asyncToGenerator.js:3:20)
    at a (asyncToGenerator.js:25:9)
instrument.ts:123 Failed connecting to device: Error: Unknown error
    at e.<anonymous> (backend.ts:176:15)
    at l (runtime.js:63:40)
    at Generator._invoke (runtime.js:294:22)
    at Generator.next (runtime.js:119:21)
    at zs (asyncToGenerator.js:3:20)
    at a (asyncToGenerator.js:25:9)
    at asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at e.<anonymous> (asyncToGenerator.js:21:12)
    at e.<anonymous> (backend.ts:93:1)
    at AdbRouter.jsx:75:38
    at l (runtime.js:63:40)
    at Generator._invoke (runtime.js:294:22)
    at Generator.next (runtime.js:119:21)
    at zs (asyncToGenerator.js:3:20)
    at a (asyncToGenerator.js:25:9)
    at asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at asyncToGenerator.js:21:12
    at AdbRouter.jsx:69:38
    at AdbRouter.jsx:135:13
    at l (runtime.js:63:40)
    at Generator._invoke (runtime.js:294:22)
    at Generator.next (runtime.js:119:21)
    at zs (asyncToGenerator.js:3:20)
    at a (asyncToGenerator.js:25:9)
    at asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at asyncToGenerator.js:21:12
    at AdbRouter.jsx:132:40
    at AdbRouter.jsx:194:7
    at l (runtime.js:63:40)
    at Generator._invoke (runtime.js:294:22)
    at Generator.next (runtime.js:119:21)
    at zs (asyncToGenerator.js:3:20)
    at a (asyncToGenerator.js:25:9)
    ```

**Comment by stylesuxx** [comment]
@sebastianwachter 

> Found Device: GP150, Version: 01.02.0000

You are not on 0606 please make sure to downgrade using the DJI Assistant.

**Comment by sebastianwachter** [comment]
I am stupid! I misread these version numbers wrong every time I launched dji assistant

![image](https://user-images.githubusercontent.com/14070927/196799644-8946d682-a98d-4439-b126-f548ff9a290c.png)

-> this fixed the problem for me - maybe this is the same for you @vruzy

**Comment by Erudite495** [comment]
Same issue here. Goggles v2 reported rooted successfully but configurator page doesn't see the port. Vers v01.00.0606 Win 10 Chrome

**Comment by stylesuxx** [comment]
I think we got @Erudite495 sorted - in their case it was a matter of changing from one USB port to another. Maybe Win 10 was doing strange stuff in this case.

**Comment by Daniel00687** [comment]
Hello,
I have the same problem.
Goggles V2
firmware version "Air-Unit-Mode" or "DIY-Mode": "v01.00.0606
firmware version "DJI-FPV-Mode" or "FPV-Racer Drone Mode": v01.02.0001
OS: Windows 10 Home
Browser: Opera and Chrome testet
also testet on another Laptop 

It says found Device: GP150, Version: 01.02.0001 
https://www.youtube.com/watch?v=hNOA0kUjKhY at 4:45 in the table he says that this combination of firmeware is possible for root hack.
is it necessary to  go with another firmeware in "DJI-FPV-Mode"? How to switch it?

Thanks!

**Comment by stylesuxx** [comment]
@Daniel00687 you need to root in DIY mode. DIY mode is lying in regards to version. Use DJI Assistant to Upgrade. Also you are saying you "have the same problem" - it seems you are not able to root, or did that work. Please always open a separate issue.

**Comment by Daniel00687** [comment]
@stylesuxx yes, after some attempts and power cycles it said Rooted successfully but the device couldn‘t be connected after rooting.
but now i did the upgrade over dji assistant to 606 and everything worked (after a few attemps). (it confused me that i could go to version 01.00.0606 after going to 01.02.0001 and the info from the youtube video that rooting with 01.02.0001 is possible)
by the way: dji assistant told me that my current version was 00.000.0000 ?? and i did an Upgrade to 606.

**Comment by stylesuxx** [comment]
If it was showing version 0000 than something else was wrong anyway. 0606 is the best version to root and install from, other version might work, but 0606 is the optimal, most tested version.

**Comment by stylesuxx** [comment]
@vruzy any update on this? Otherwise I will close the issue.

**Comment by vruzy** [comment]
no updates. Never got it to work and haven’t continued experimenting or trying anything new.

Shaun Ortman
***@***.***
(216)307-4286
www.treestreetstudio.com


On Nov 29, 2022, at 7:00 AM, Chris ***@***.***> wrote:

﻿

@vruzy<https://github.com/vruzy> any update on this? Otherwise I will close the issue.

**Comment by jstoeffler** [comment]
Hi everyone,

I had the same problem, and I managed to fix it, so I thought I'd share what I tried.

First my setup:

- Mac OS Intel processor
- Chrome
- DJI FPV Goggles V1 V01.00.0608 (it's weird it seems everyone acts like V01.00.0606 is the last version, so I was wondering if something was wrong about this)

And as for the thing that solved it, I tried many things at the same time, so I am not sure, but the last things I tried were:

1. Using https://develop.fpv.wtf/ On my machine this version had problems with CORS, and I bypassed these using the `--disable-web-security` and `--user-data-dir flags=...` flahs. Not sure how this works on windows, but on mac it was something like: `/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --disable-web-security --user-data-dir flags=~/Desktop/some-folder/`
2. Check the USB connection. I used another brand new USB cable, and I made sure it was well connected, and that there was no physical constraint on it.

I hope this helps.
